src/tests/edge_ids/transition_id_tests.rs

```diff
@@ -0,0 +1,96 @@
+        QueryExpression::Conjunction(
+            Box::new(QueryExpression::VarName("HalfAdm1".to_string())),
+            Box::new(QueryExpression::VarName("HalfAdm2".to_string()))),
+        vec![
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E43".to_string())),
+                Box::new(TransitionID::Simple("E31".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E37".to_string())),
+                Box::new(TransitionID::Simple("E34".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E42".to_string())),
+                Box::new(TransitionID::Simple("E33".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E37".to_string())),
+                Box::new(TransitionID::Simple("E35".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E42".to_string())),
+                Box::new(TransitionID::Simple("E30".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E39".to_string())),
+                Box::new(TransitionID::Simple("E31".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E38".to_string())),
+                Box::new(TransitionID::Simple("E32".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E41".to_string())),
+                Box::new(TransitionID::Simple("E34".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E40".to_string())),
+                Box::new(TransitionID::Simple("E33".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E40".to_string())),
+                Box::new(TransitionID::Simple("E30".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E38".to_string())),
+                Box::new(TransitionID::Simple("E36".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E41".to_string())),
+                Box::new(TransitionID::Simple("E35".to_string()))
+            )
+            ]; "Conjunction HalfAdm1 and HalfAdm2")]
+    fn transition_id_checker(
+        path: &str,
+        machineExpression: QueryExpression,
+        transition_ids: Vec<TransitionID>,
+    ) {
+        let mock_model = Box::new(machineExpression);
+        let mut expected_ids: HashSet<&TransitionID> = HashSet::from_iter(transition_ids.iter());
+        let (_, system) =
+            reachability_test_helper_functions::create_system_recipe_and_machine(*mock_model, path);
```

Comment:
synes der skal stå FOLDER_PATH istedet for path

Reply:
doen